src/pyflex/config.py

```diff
@@ -23,8 +23,10 @@ class Config(object):
     def __init__(self, min_period, max_period, stalta_waterlevel=0.07,
                  tshift_acceptance_level=10.0, tshift_reference=0.0,
                  dlna_acceptance_level=1.3, dlna_reference=0.0,
-                 cc_acceptance_level=0.7, s2n_limit=1.5, earth_model="ak135",
+                 cc_acceptance_level=0.7, earth_model="ak135",
+                 s2n_limit_energy=2.0, s2n_limit_amplitude=1.5,
```

Comment:
What about keep the old name "s2n_limit" and add the new variable "s2n_limit_energy". So people don't need to change there existing scripts?